subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -10,6 +10,7 @@ struct lll_conn_iso_stream_rxtx {
 	uint64_t max_pdu:8;        /* Maximum PDU size */
 	uint64_t ft:8;             /* Flush timeout (FT) */
 	uint64_t bn:4;             /* Burst number (BN) */
+	uint64_t bn_curr:4;        /* Current burst number */
```

Comment:
Move it after `rfu:1` so that the 64-bits remain as before, and use `uint8_t bn_curr:1;`